scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
The problem is that if we don't loop through frameworks like this, and just call build, the first build failure (for net451) causes the other tfm builds not to happen and we get empty packages.

It is fairly ugly and needs cleanup I agree, but it is outside the scope of this PR.